eligibility_api/api.py

```diff
@@ -139,28 +139,36 @@ def __init__(
 class Client:
     """Eligibility Verification API HTTP client."""
 
-    def __init__(self, agency, verifier, issuer):
-        logger.debug(f"Initialize client for agency: {agency.short_name}")
-        self.issuer = issuer
-
-        # get the eligibility type names
-        self.types = list(map(lambda t: t.name, agency.types_to_verify()))
-        self.agency_id = agency.agency_id
-        self.jws_signing_alg = agency.jws_signing_alg
-        self.client_private_jwk = agency.private_jwk
-        self.jwe_encryption_alg = verifier.jwe_encryption_alg
-        self.jwe_cek_enc = verifier.jwe_cek_enc
-        self.server_public_jwk = verifier.public_jwk
-
-        self.api_url = verifier.api_url
-        self.api_auth_header = verifier.api_auth_header
-        self.api_auth_key = verifier.api_auth_key
+    def __init__(
+        self,
+        api_url,
+        api_auth_header,
+        api_auth_key,
+        issuer,
+        agency_identifier,
+        jws_signing_alg,
+        client_private_jwk,
+        jwe_encryption_alg,
+        jwe_cek_enc,
+        server_public_jwk,
+    ):
+        self.api_url = api_url
+        self.api_auth_header = api_auth_header
+        self.api_auth_key = api_auth_key
 
-    def _tokenize_request(self, sub, name):
+        self.issuer = issuer
+        self.agency_identifier = agency_identifier
+        self.jws_signing_alg = jws_signing_alg
+        self.client_private_jwk = client_private_jwk
+        self.jwe_encryption_alg = jwe_encryption_alg
+        self.jwe_cek_enc = jwe_cek_enc
+        self.server_public_jwk = server_public_jwk
+
+    def _tokenize_request(self, sub, name, types):
         """Create a request token."""
         return RequestToken(
-            self.types,
-            self.agency_id,
+            types,
+            self.agency_identifier,
```

Comment:
Renamed this to be clearer that it is the [field](https://docs.calitp.org/eligibility-api/specification/#request) from the API as opposed to the `id` from benefit's `TransitAgency`.

Reply:
I have a thought about this, left a review comment. Remember benefit's `TransitAgency` doesn't exist from this repo's point of view.